spec/acceptance/nodesets/debian-9.yml

```diff
@@ -0,0 +1,15 @@
+HOSTS:
```

Comment:
In Voxpupuli and Theforeman we generate this on the fly: `PUPPET_INSTALL_TYPE=agent BEAKER_IS_PE=no BEAKER_PUPPET_COLLECTION=puppet5 BEAKER_debug=true BEAKER_setfile=centos7-64{hypervisor=docker\,hostname=centos7-64.example.com} bundle exec rake beaker`

You can use `debian9-64` instead of `centos7-64` or `ubuntu1604-64`. The hostname isn't needed but some applications prefer a FQDN instead of a short name.

Reply:
Ah, I wasn't aware of that possibility, that's very nice.

Reply:
I added it to beaker-hostgenerator in https://github.com/puppetlabs/beaker-hostgenerator/commit/5adf569c1a0d8cf1071c53ccda7bea0fbecde30a so the various projects could easily reuse work from others.

Reply:
@ekohl nice work, I'm testing it right now.